packages/core/src/components/radio-pill/_radio-pill.scss

```diff
@@ -0,0 +1,56 @@
+@include exports('ray-radio-pill') {
+  $ray-radio-pill-border-width: 1px;
+  $ray-radio-pill-border-radius: 4px;
```

Comment:
```suggestion
  $ray-radio-pill-border-radius: rem(4px);
```

Reply:
changing this to use the global `ray-border-radius`